jinahub/segmenters/VADSpeechSegmenter/vad_speech_segmenter.py

```diff
@@ -0,0 +1,140 @@
+__copyright__ = 'Copyright (c) 2020-2021 Jina AI Limited. All rights reserved.'
+__license__ = 'Apache-2.0'
+
+from pathlib import Path
+from typing import Optional, Tuple
+
+import torch
+import torchaudio
+import numpy as np
+import librosa as lr
+from jina import Executor, Document, DocumentArray, requests
+from jina.excepts import BadDocType
+
+
+class VADSpeechSegmenter(Executor):
```

Comment:
Shouldn't we give the option to specify which device the model should be on? We have it in other pytorch executors

Reply:
The Silero does offer a couple of different models. However, each of them has different functions and will return different results. As this executor is only for detecting speech,  only the `sliero_vad` model is fit for this executor and other Silero models would be incompatible. Since there are no other options, we can remove `model` from the argument.